File: Lenguajes Orientados a Objetos/Actividad 9/python/empleadoTest.py
from empleado import Empleado
empl_void = Empleado()
empl_void.Imprimir()
empl_void.calcularSalario()

empl = Empleado("Donald Trump", 2025, 200000)


while True: 
    try: 

        nombre = input("Introduzca el nombre del empleado: ")
        anio = int(input("Introduzca el anio de contratacion del empleado: "))
        salario = float(input("Introduzca el salario del empleado: "))
        break
    except ValueError:
        print("Introduzca un valor adecuado.")

empl_manual = Empleado(nombre, anio, salario)

File: Lenguajes Orientados a Objetos/Actividad 9/Cpp/empleadoTest.cpp
#include <string>
#include "empleado.h"
#include <iostream>

using namespace std;

int main()
{
    Empleado empl_void;
    empl_void.Imprimir();
    empl_void.calcularSalario();

    Empleado empl("Donald Trump", 2025, 20000);
    empl.Imprimir();
    empl.calcularSalario();


    string nombre;
    int anio;
    double salario;

    cout << "Introduzca el nombre del empleado: ";
    cin >> nombre;
    cout << "\nIntroduzca el anio en que se contrato: ";
    cin >> anio;
    cout << "\n Introduzca el salario mensual de empleado: ";
    cin >> salario;
    cout << endl; 


    Empleado empl_manual(nombre, anio, salario);

    empl_manual.Imprimir();
    empl_manual.calcularSalario();
    

    return 0;
}